api/internal/target/kusttarget.go

```diff
@@ -119,7 +119,7 @@ func (kt *KustTarget) MakeCustomizedResMap() (resmap.ResMap, error) {
 
 func (kt *KustTarget) makeCustomizedResMap() (resmap.ResMap, error) {
```

Comment:
Would it make sense to change `origin` as a KustTarget attribute?
Reason:
- `origin` is more of an attribute than an argument. attribute is a struct/class feature or holds struct/class info, here origin ties with the kustomization/kustTarget); argument provides key input to a function and normally change/modify the function workflow.   
- It will simplify the code since  KustTarget no longer needs to pass `origin` to its functions like `runTransformers`, `AccumulateTarget` `configureBuiltinTransformers` `configureExternalTransformers`.

Reply:
I think it makes sense. I am trying to weigh in my head if intuitively `origin` makes sense as part of KustTarget, given that the description of KustTarget is `KustTarget encapsulates the entirety of a kustomization build.` From that perspective, the `origin` object is keeping track where we are in the filesystem (or a repo) in our current run of the build, so yes, I think it would make sense to include as an attribute, especially since the `origin` is following the KustTarget throughout the entirety of the build. 

I'll update the PR accordingly.

Reply:
done!

Reply:
looks better!